syntaxes/KCL_mod.tmLanguage.json

```diff
@@ -0,0 +1,106 @@
+{
+	"$schema": "https://raw.githubusercontent.com/martinring/tmlanguage/master/tmlanguage.json",
+	"name": "KCL_mod",
+	"patterns": [
```

Comment:
These patterns are from KCL and do not match in the mod file. You can make some corresponding modifications, for example, keywords include `package`, `dependencies` instead of `as` `assert` and the like